packages/graphql/src/schemaTypes/objectTypes/gql-GitInfo.ts

```diff
@@ -0,0 +1,19 @@
+import { objectType } from 'nexus'
+
+export const GitInfo = objectType({
```

Comment:
This is subjective but I like the name `Git` over `GitInfo`. I think the Info is implied by it giving us data and Git lines up with our other nodes like `Project` and `Spec`

Reply:
I think `GitInfo` makes sense for this, or even `SpecGitInfo`. I see what you're getting at @ImCesar , but `Project` / `Spec` are first class data models of the app, but `Git` makes it sound like it'd be a more generic interface to exposing data from the `git`:

```graphql
git {
   lastCommit
   untrackedFileCount
   isDirty
   currentBranch
}
```